### ISSUE TITLE
ArcMap 10.6 crashes in arcbrutile.dll when trying to delete a file from the ArcBruTile directory.

### ISSUE THREAD

**Issue by poiprotocol** [ArcBruTile/ArcBruTile#21]
ArcMap 10.6 crashes with an unhandled exception in arcbrutile.dll when trying to delete a file from the ArcBruTile directory.

**Exception type:**   System.IO.IOException
**Message:**    `The process cannot access the file '%AppData%\Roaming\ArcBruTile\<redacted>\<redacted>\<redacted>\<redacted>.png' because it is being used by another process.`

```
StackTrace (generated):
    SP       IP       Function
    26E0F1F8 6C5246AA mscorlib_ni!System.IO.__Error.WinIOError(Int32, System.String)+0xaaac5a
    26E0F230 6C57BE89 mscorlib_ni!System.IO.File.InternalDelete(System.String, Boolean)+0xa456ad
    26E0F248 6BB367D6 mscorlib_ni!System.IO.File.Delete(System.String)+0x16
    26E0F254 2A400448 arcbrutile.dll!Unknown+0x198
    26E0F2B8 6BAAAB43 mscorlib_ni!System.Threading.ThreadHelper.ThreadStart_Context(System.Object)+0x6f
```

**Comment by bertt** [comment]
How are you deleting the tiles? Using File.Delete in a ArcMap command? Whats the uscase?

**Comment by bertt** [comment]
added tile delete exception handling, fixed in 1.0.3
